cpp/ql/test/library-tests/ir/ir/aliased_ssa_consistency.expected

```diff
@@ -21,6 +25,14 @@ useNotDominatedByDefinition
 switchInstructionWithoutDefaultEdge
 notMarkedAsConflated
 wronglyMarkedAsConflated
+| ir.cpp:1038:12:1038:18 | Chi: [...](...){...} | Instruction 'Chi: [...](...){...}' should not be marked as having a conflated result in function '$@'. | ir.cpp:1038:6:1038:8 | (lambda [] type at line 1038, col. 12) lam | (lambda [] type at line 1038, col. 12) lam |
+| ir.cpp:1038:12:1038:18 | Chi: [...](...){...} | Instruction 'Chi: [...](...){...}' should not be marked as having a conflated result in function '$@'. | ir.cpp:1038:6:1038:8 | (lambda [] type at line 1038, col. 12) lam | (lambda [] type at line 1038, col. 12) lam |
+| ir.cpp:1695:16:1695:16 | Chi: 1 | Instruction 'Chi: 1' should not be marked as having a conflated result in function '$@'. | ir.cpp:1695:5:1695:12 | int global_2 | int global_2 |
```

Comment:
What's causing these failures? The ones in lambdas don't concern me that much, because we know we have issues there, but the ones in initializers should at least be investigated for this PR, even if the outcome is just a separate issue with a repro case.

Reply:
Not entirely sure yet - there's Chi instructions being inserted for the global variable, and those are being marked as conflated. I'm not sure the Chis should be there in the first place,

Reply:
The Chis are generated and marked as conflated because the global variable is defined to escape, and that's a consistency failure because the chi chain doesn't go back to an `AliasedDefinitionInstruction`.

Reply:
I think the easiest fix here is to consider the variable not to escape in its own initializer, but I'm a little nervous about having no aliased memory instruction in more complicated initializers. What do you think?

Reply:
On second thought adding the aliased memory isn't that hard, so I've gone ahead and done it